templates/search/results.html.tx

```diff
@@ -100,37 +100,49 @@
 </article><!-- search-container -->
 
 % if $pager.show_pager {
-    <div class="search-meta" id="search-meta">
-        <ul class="pager" id="pager">
+        <nav class="govuk-!-margin-top-6 govuk-!-margin-bottom-2 govuk-pagination" aria-label="Pagination">
             % if ($pager.previous_page) {
-                <li>
-                    <a class="page govuk-link govuk-pagination__link" id="previous-page" href="<% $c.url_with.query([ page => $pager.previous_page ]) %>">
-                        Previous
+                <div class="govuk-pagination__prev">
+                    <a class="govuk-link govuk-pagination__link" href="<% $c.url_with.query([ page => $pager.previous_page ]) %>" rel="prev">
+                    <svg class="govuk-pagination__icon govuk-pagination__icon--prev" xmlns="http://www.w3.org/2000/svg" height="13" width="15" aria-hidden="true" focusable="false" viewBox="0 0 15 13">
+                        <path d="m6.5938-0.0078125-6.7266 6.7266 6.7441 6.4062 1.377-1.449-4.1856-3.9768h12.896v-2h-12.984l4.2931-4.293-1.414-1.414z"></path>
+                    </svg>
+                        <span class="govuk-pagination__link-title">
+                            Previous<span class="govuk-visually-hidden"> page</span>
+                        </span>
                     </a>
-                </li>
+                </div>
             % }
+            <ul class="govuk-pagination__list">
             % for $pager.pages_in_set -> $page {
-                <li>
                     % if $page <= $pager.page_limit {
                         % if $pager.current_page == $page {
-                            <span class="govuk-visually-hidden">Page </span>
```

Comment:
I think here you could simplify this section  a bit:

```html
<li class="govuk-pagination__item">
    <a
        class="govuk-link govuk-pagination__link <% if $pager.current_page == $page { %> govuk-pagination__item--current <% } %>"
        href="<% $c.url_with.query([ page => $page ]) %>"
        aria-label="Page <% $page %>"
        % if $pager.current_page == $page {
            aria-current="page"
        % }
    >
         <% $page %>
    </a>
</li>
```

Note that this also includes the updated `aria-label` and `aria-current` attributes which are important for accessibility.